shared/firestore-models/util.ts

```diff
@@ -13,15 +13,25 @@ export async function isConfiguredTargetLabel(configId: string, label: string) {
   return false;
 }
 
-/** Determine if a pull request as one of the targeted labels based on the repos config. */
-export async function hasTargetedLabel(
+/** Retrieve the target branch for a pull request based on the repos config. */
+export async function getTargetBranch(
```

Comment:
Add an explicit return type for this function? It's not super obvious to me what it should be from the code.